src/index.js

```diff
@@ -59,6 +59,7 @@ export { reprojectTexture } from './graphics/reproject-texture.js';
 export { programlib } from './graphics/program-lib/program-lib.js';
 export { shaderChunks } from './graphics/program-lib/chunks/chunks.js';
 export { GraphicsDevice } from './graphics/graphics-device.js';
+export { Prefilter } from './graphics/prefilter.js';
```

Comment:
this could be alphabetically sorted.

Reply:
It kind of already is if you do 'a..z' then 'A..Z'. 😄